src/Core/Form/IdentifiableObject/DataProvider/ProductFormDataProvider.php

```diff
@@ -280,6 +285,37 @@ private function extractRelatedProducts(int $productId): array
         return $relatedProductsData;
     }
 
+    /**
+     * @param int $productId
+     *
+     * @return array<int, array<string, int|string>>
+     */
+    private function extractPackedProducts(int $productId): array
+    {
+        /** @var PackedProductDetails[] $packedProductsDetails
+         */
+        $packedProductsDetails = $this->queryBus->handle(
+            new GetPackedProductsDetails(
+                new ProductId($productId),
+                new LanguageId($this->contextLangId)
+            )
+        );
+        $packedProductsData = [];
+        foreach ($packedProductsDetails as $packedProductDetails) {
+            $productName = $packedProductDetails->getProductName();
+            $packedProductsData[] = [
+                'productId' => $packedProductDetails->getProductId(),
+                'name' => $productName,
+                'combinationId' => $packedProductDetails->getCombinationId(),
```

Comment:
@zuk3975 is right we use snake case convention for form fields

Reply:
ok I refactor to push only snake case to front and keep CamelCase in back side